pages/contact.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <slot name="contact-thankyou">
+        <div v-if="thankyouMessage" class="message">
+          <div class="message__container">
+            <div class="icon-wrapper">
+              <icon icon="ic:outline-email" class="icon-wrapper__icon"> </icon>
+            </div>
+            <div class="text-content">
+              {{ $t('contact.form_sent.thankyou_title') }}
+              <div>
+              {{ $t('contact.form_sent.thankyou_description') }}
+              </div>  
+            </div>
+            <a href="/" class="btn-outline btn-primary btn-wide btn">
+                {{ $t('btn.back') }}
+              </a>
+          </div>
+        </div>
+      </slot> 
+  <contact-form v-if="!thankyouMessage" @lead="sendLead" dataPolicyPage="/data-protection"></contact-form>
+</template>
+<script lang="ts">
+import { Lead } from '~/models/Lead'
+import ContactForm  from "~/components/global/ContactForm.vue";
+
+export default defineNuxtComponent({
+  components: {
+    'contact-form': ContactForm
+  },
+  data(){
+    return {
+      thankyouMessage: false
+    }
+  },
+  methods: {
+    async sendLead(lead: Lead) {
+      const leadsService = useShopinvaderService('leads')
```

Comment:
I'm not sure to understand why it's better to call API request here instead of directly in the component

Reply:
not sure either, but we can imagine all the websites will have a contact page. I can also refactor it to be taken into account on the component.